osmosis-1/osmosis.zone_assets.json

```diff
@@ -2806,6 +2807,13 @@
       "osmosis_verified": true,
       "osmosis_unlisted": true,
       "listing_date_time_utc": "2024-03-12T12:00:00Z"
+    },
+    {
+      "chain_name": "odin",
+      "base_denom": "udoki",
+      "path": "transfer/channel-258/udoki",
+      "twitter_URL": "https://twitter.com/doki_coin",
```

Comment:
Could you please Register this (and a website if there is one) to the Cosmos Chain Registry?
It is not required for listing, but it is become part of the 'Verified' status upgrade process.